src/config.js

```diff
@@ -0,0 +1,201 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+/**
+ * Handles a configuration dictionary.
+ *
+ * @class Config
+ * @extends Model
+ */
+
+CKEDITOR.define( [ 'ckeditor', 'mvc/model', 'utils' ], function( CKEDITOR, Model, utils ) {
```

Comment:
Ops, true... leftover from a previous implementation :)

It's weird that jshint doesn't catch this... maybe because `CKEDITOR` is configured as a global, still it is passed as parameter here, which overrides the global :/